### DISCUSSION TITLE
Customize existing database tables

### DISCUSSION THREAD

**Discussion by callumagmo** [medusajs/medusa#1160] [Q&A] [answered]
How can we modify existing database tables to cope with our needs?
For example, adding the custom fields in the User table, and updating the GET, POST, PUT user APIs.

Understand that the medusa is installed as a package, so we cant modify it directly. 
Does it mean that we need to fork the repo and modify it?

Thanks in advance!!

**Comment by srindom** [comment, marked as answer]
Hi Callum! 

Sounds like you might be able to leverage this awesome community project built by @adrien2p to accomplish your needs: https://github.com/adrien2p/medusa-extender

For most use cases it is recommended that you don't make too many modifications to the core db entities in order to make it easy to upgrade packages in the future, but there is always the option of extending further through the extender project if required. 

Would love to hear more about your use case - it might be that it would make sense for us to extend the core schema to allow for what you need :)

**Reply by SyedYaseen** [reply]
Thats archived now.
